components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/dto/AppCallTypeDTO.java

```diff
@@ -9,28 +10,45 @@
 
 
     private String appName;
-    private String apiName;
-    private String  consumerKey;
-    private List<String> methods;
+    List<ApiCallTypeArray> apiCallTypeArray=new ArrayList<ApiCallTypeArray>();
 
+    public String getAppName() {
+        return appName;
+    }
 
-    public String getconsumerKey() {return consumerKey; }
-
-    public void setconsumerKey(String consumerKey) { this.consumerKey = consumerKey;}
-
-    public String getappName() {return appName; }
+    public void setAppName(String appName) {
+        this.appName = appName;
+    }
 
-    public void setappName(String appName) { this.appName = appName;}
+    public List<ApiCallTypeArray> getApiCallTypeArray() {
+        return apiCallTypeArray;
+    }
 
-    public List<String>  getCallType() {return methods;}
+    public void addGToApiCallTypeArray(String apiName, List<String> callType) {
+        ApiCallTypeArray a=new ApiCallTypeArray();
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea